web/src/locales/zh-CN/component.ts

```diff
@@ -21,19 +21,24 @@ export default {
   'component.global.search': '查询',
   'component.global.reset': '重置',
   'component.global.confirm': '确认',
+  'component.global.format': '格式',
```

Comment:
I remember I have reviewed one PR like this 😳 Why will I see it again?

Reply:
idk i guess there is some commit issue.

Reply:
haha, let's review one by one  currently.

Reply:
Yes yes